### PR TITLE
Streamline the challenge

### PR DESCRIPTION
**Pull request by SiAdcock**
## Removes requirement on http-server

This seems like unnecessary bloat that doesn't help the candidate complete this task. It also contradicts the "Client side only" requirement, which confuses things.

## Removed mandatory / optional requirements

I've instead given a brief description of the problem and left the optional requirements as "considerations" (choice of wording here TBD!) which the candidate can choose to focus on.

## More explanation of the structure of the process

I've been more explicit about how the candidate will be invited in to the Guardian to discuss their choices. I've prompted them to highlight their choices in a README.

## No frameworks rule

I've explicitly banned Angular and any CSS framework, as they make it possible to complete the challenge without writing much application-specific code. The purpose of the challenge is to encourage the candidate to demonstrate their coding approach and structure, not how quickly they can stitch together existing libraries that meet the requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,45 @@
...
 
 #### Browser support
 
-Our site uses modern web standards to give the best possible reading experience for our visitors. Although we aim to be readable on all possible devices and browsers, not all are capable of running the full set of features. We'd like you to make sure your application works on older browsers as well as the latest browsers. Our [reccommended browser list](https://www.theguardian.com/help/recommended-browsers) states the browsers we currently support.
+Our site uses modern web standards to provide the best possible reading experience for our visitors. Although we aim to be readable on all possible devices and browsers, not all are capable of running the full set of features. We'd like you to make sure your application works on older browsers as well as the latest browsers. Our [recommended browser list](https://www.theguardian.com/help/recommended-browsers) states the browsers we currently support.
 
 #### No JavaScript
```
Comment (SiAdcock) [inline README.md]: No sure what to do about this consideration. I don't see how we can offer "a meaningful experience when JavaScript is unavailable" without a server side requirement

File: README.md [hunk 2]
```diff
@@ -1,56 +1,45 @@
...
-Inside this repo is an incomplete **tabbed component**. We'd like you to pick up this piece of work and update it so it meets the requirements listed below.
-
-## Requirements
-
-The requirements have been split into manadatory and optional groups. Please take time to read the requirements carefully. We'd like you to have a go at achieving the mandatory requirements and if you have the time you can then choose 2 optional requirements to focus on.
-
-### Mandatory requirements
+Inside this repo is an incomplete **tabbed component**. We'd like you to update it to meet the requirements listed below. 
 
 You should implement a **tabbed component** that looks similar to the Guardian’s most recent content widget. The component will be a numbered listing of the most recent Guardian content chosen from three sections of the site: UK news, Football and Travel. Each of these sections should be represented by a tab.
```
Comment (SiAdcock) [inline README.md]: Should we add a screenshot / wireframe of the sort of thing we expect?

File: README.md [hunk 3]
```diff
@@ -1,56 +1,45 @@
...
 
-We’d like you to fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). Please do this on the client-side, your solution should run in the browser alone with no server-side requirements.
+You should fetch this data asynchronously from our [Content API](https://open-platform.theguardian.com/explore/). **Please do this on the client-side.** Your solution should run in the browser alone with no server-side requirements.
```
Comment (SiAdcock) [inline README.md]: Perhaps this wording need clarification that **we are not ruling out build tooling**. What do you think?

File: README.md [hunk 4]
```diff
@@ -1,65 +1,54 @@
...
-Inside this repo is an incomplete **tabbed component**. We'd like you to pick up this piece of work and update it so it meets the requirements listed below.
-
-## Requirements
-
-The requirements have been split into manadatory and optional groups. Please take time to read the requirements carefully. We'd like you to have a go at achieving the mandatory requirements and if you have the time you can then choose 2 optional requirements to focus on.
-
-### Mandatory requirements
+Inside this repo is an incomplete **tabbed component**. We'd like you to update it to meet the requirements listed below. 
 
 You should implement a **tabbed component** that looks similar to the Guardian’s most recent content widget. The component will be a numbered listing of the most recent Guardian content chosen from three sections of the site: UK news, Football and Travel. Each of these sections should be represented by a tab.
```
Comment (akash1810) [inline README.md]: I wonder if we should add links to the CAPI queries as I don't think assessing a candidate's ability to read our documentation is necessary?

File: README.md (hunk 3 above)
Comment (akash1810) [reply README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -1,65 +1,54 @@
...
+At the Guardian we try to build things to be lightweight, accessible, and usable across browsers, devices and network conditions. 
 
-### Optional requirements
+During implemetation, please choose 2 or 3 of the below considerations to focus on. Let us know which ones you have chosen in your repo's README.
 
-#### Accessibility
+### Accessibility
 
-A great deal of web content can be made accessible just by making sure the correct HTML elements are used for the correct purpose at all times. We'd like you consider how the HTML can be used to ensure maximum accessibility for our user's that rely on screen reading technology and/or keyboard navigation to access our content.
+A great deal of web content can be made accessible simply by using semantically correct HTML elements and role attributes. We'd like you to ensure the component is accessible for our users that rely on screen reading technology or keyboard navigation to access our content.
```
Comment (akash1810) [inline README.md]: users ➡️ readers?

File: README.md [hunk 6]
```diff
@@ -1,65 +1,54 @@
...
-At the Guardian we try to build things to be lightweight, accessible, and usable across browsers, devices and network conditions. When working on your application you should therefore focus on:
+## Considerations
 
-- Keeping the application as lightweight as possible.
-- Making it work across a range of device sizes.
-- Using semantically correct HTML.
+At the Guardian we try to build things to be lightweight, accessible, and usable across browsers, devices and network conditions. 
 
-### Optional requirements
+During implemetation, please choose 2 or 3 of the below considerations to focus on. Let us know which ones you have chosen in your repo's README.
```
Comment (akash1810) [inline README.md]: should we also say all 5 points will for a discussion in the review stage?

File: README.md (hunk 4 above)
Comment (SiAdcock) [reply README.md]: The [API docs here](https://open-platform.theguardian.com/explore/) are quite easy to follow though, as long as you don't want to do anything exotic like show a field that isn't included by default! 😱 I have heard complaints from previous candidates that they couldn't get the `trailText` field from CAPI, but this is no longer a requirement.

Shall we leave it as it is until we hear something negative from a candidate?

File: README.md (hunk 1 above)
Comment (GHaberis) [reply README.md]: Agreed, it seems bit much expecting someone to implement a server side solution in the event of no-js within the time limit. The requirement could be alluding to using some kind of CSS fallback for tabs _if_ the tabs list had been server side rendered and no-js was running in the browser? But it's difficult to describe that without it being confusing.

File: README.md (hunk 2 above)
Comment (GHaberis) [reply README.md]: Yes, a screenshot would be good!

File: README.md (hunk 4 above)
Comment (GHaberis) [reply README.md]: I'd say keep it as it is, I don't know of candidates having struggled to get the relevant information out of the API with the info provided, and yes it's good we're no longer suggesting using fields such as `trailText` that require special/undocumented API requests!

**Review by GHaberis — APPROVED** [review]
Looks great 👍

**Pull request merged**